packages/gatsby-theme-store/src/gatsby-node.ts

```diff
@@ -209,6 +222,14 @@ export const onCreateWebpackConfig = (
     resolve: {
       alias: {
         ...profilingConfig?.resolve.alias,
+        'gatsby-plugin-next-seo$': resolve(
```

Comment:
I think it'd be good for this magic to be explained--both the reason for this and what it does.

Reply:
this comment is for the whole webpack magic, not just for this line